assets/js/swap-phrase.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+(function() {
```

Comment:
Probably we don't need this since we are using `ES6 import` here.

More info: https://hashnode.com/post/do-es6-modules-make-the-case-of-iifes-obsolete-civ96wet80scqgc538un20es0